package.json

```diff
@@ -4,7 +4,7 @@
   "description": "A presentation augmentation app that gives teachers and presenters real-time feedback on the effectiveness of their lecture.",
   "main": "index.js",
   "scripts": {
-    "test": "./node_modules/.bin/eslint **/*.js **/*.jsx | ./node_modules/mocha/bin/mocha --compilers js:babel/register",
+    "test": "./node_modules/.bin/eslint **/*.js **/*.jsx || ./node_modules/mocha/bin/mocha --compilers js:babel/register",
```

Comment:
Single pipe in BASH literally puts the output of the first command into the second, meaning we don't see the results of the linter.

If you want to run two commands in one line in BASH, you either use ';'  if you want them to run independently, or you put '||' meaning you want them both to run, even if the first fails.  

</pedant>